src/WalkingModule/app/robots/iCubGazeboV2_5/dcm_walking_with_joypad.ini

```diff
@@ -19,6 +19,10 @@ com_height              0.53
 # sampling time
 sampling_time           0.01
 
+# comment out the following line if the position of the base is not provided by an
+# external software(here Gazebo)
+use_external_robot_base            1
```

Comment:
The same here, I am not sure if this option should be on by default. I would rather prefer having in simulation a realistic scenario. You can modify it locally, but I would prefer this to be off by default.